src/Features/Core/Portable/QuickInfo/QuickInfoItem.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.QuickInfo
+{
+    internal sealed class QuickInfoItem
+    {
+        /// <summary>
+        /// The span of the document that the item is based on.
+        /// </summary>
+        public TextSpan Span { get; set; }
+
+        /// <summary>
+        /// Descriptive tags from the <see cref="Microsoft.CodeAnalysis.Completion.CompletionTags"/> type.
+        /// These tags may influence how the item is displayed.
+        /// </summary>
+        public ImmutableArray<string> Tags { get; }
```

Comment:
Do we have examples of what sort of tags we use here?  Is this to give things like the icon displayed in QuickInfo?   Or is that information within the TextBlocks?